tox.ini

```diff
@@ -1,10 +1,17 @@
 [tox]
-envlist = py{37,38,39,310,311}-dj{32,40,41,main}, lint
+envlist =
+    py{37,38,39,310}-dj32
+    py{38,39,310,311}-dj40
```

Comment:
it is a good time to drop Django 4.0 from the matrix

Reply:
👍 I’ll do that and Python 3.7 separately.